#!/bin/python
PACKAGE = "cr_dai_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_amp_param", double_t, 0, "min amplitude",    20, 0,   1000)
gen.add("max_error_param", double_t, 0, "max error",    .1, 0,   1)
gen.add("min_dist_param", double_t, 0, "min distance",    0, 0,   10)
gen.add("max_dist_param", double_t, 0, "max distance",    5, 0,   10)
gen.add("max_albedo_param", double_t, 0, "max albedo",    2000, 0,   5000)
gen.add("min_albedo_param", double_t, 0, "min albedo",    30, 0,   5000)
gen.add("albedo_cutoff_amp", double_t, 0, "albedo_cutoff_amp",    50, 0,   1000)
gen.add("max_asymmetry", double_t, 0, "asymmetry tolerance", 0, 0, 100)
gen.add("use_loaded_filter", bool_t, 0, "Use loaded filter",    True)

exit(gen.generate(PACKAGE, "cr_dai_ros_msgs", "ToFControl"))
